frontend/survey-project/src/styles.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["styles.scss","styles.css"],"names":[],"mappings":"AAAA;EACI,wBAAA;EACA,0BAAA;EACA,wBAAA;EACA,uBAAA;EACA,qBAAA;EACA,mBAAA;EACA,mBAAA;EACA,sBAAA;EAEA,yCAAA;EACA,2CAAA;ACAJ;;ADGA;EACI,8CAAA;EACA,8CAAA;ACAJ","file":"styles.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.